src/ecdsa/test_pyecdsa.py

```diff
@@ -1204,8 +1204,6 @@ def do_test_to_openssl(self, curve, hash_name="SHA1"):
 
 
 class TooSmallCurve(unittest.TestCase):
-    @pytest.mark.skipif("prime192v1" not in OPENSSL_SUPPORTED_CURVES,
```

Comment:
it is not unnecessary, there are widely deployed versions of OpenSSL that support only P-256, P-384 and P-521 curves, not to mention that openssl may be compiled without support for ECC in the first place